src/editor/components/controls.tsx

```diff
@@ -20,11 +69,12 @@ export const ToggleMarkButton: React.FC<ToggleMarkButtonProps> = (
 ) => {
   const editor = useSlate();
   const active = isMarkActive(editor, props.mark);
-  const className = active ? "editor-control active" : "editor-control";
+  const className = active ? `editor-control active` : "editor-control";
```

Comment:
Would be nice if there could be some indeterminate state for the dropdown buttons that is activated when a dropdown item is active (so for example, the emphasis button would show this indeterminate state if emphasis 4 is selected).

Reply:
I've done this for the `EditorDropdown` here: https://github.com/UWmathNEWS/slugline-web/pull/68/commits/1cb67d5d34e0d49fe4ca701ac1151b41aac8b88e, but I'm not quite sure how this would work for the 
"More" dropdown considering its actually one button and not two like the others are. 

Just inverting the colors like we do for the formatting buttons looks kind of ugly in my opinion.